### PR TITLE
Feature/1 formatter and linter

### PR DESCRIPTION
**Pull request by dedenbangkit**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 79
```
Comment (kjkoster) [inline setup.cfg:2]: I use 80, not 79. As you like.

**Review by kjkoster — APPROVED** [review]

**Comment by dedenbangkit** [comment]
Hi @kjkoster, Please note that the pre-commit configuration is not perfect, so some files might need to be fixed manually. Here's an example issue when I try to modify a file:

Screenshot from 2024-05-08 15-58-30

Please merge if it's ok for you.

**Comment by kjkoster** [comment]
Archivist is broken at this point. I only committed it, so it would not get lost in a code swap. I might delete it entirely.

**Comment by kjkoster** [comment]
@dedenbangkit looks like your commits were not signed?

**Pull request merged**